src/site/_static/css/custom.css

```diff
@@ -1,14 +1,13 @@
 /*
     Title and background colors for the class activity.
 */
-.rst-content .activity .admonition-title {
+/*  .rst-content .activity .admonition-title {
     background: #5594ca
 }
  
 .rst-content .activity {
     background: #c5dfff
-}
-
+} */
```

Comment:
Don't comment code out, delete it. Let version control do it's job.

Reply:
:+!:

Reply:
> Don't comment code out, delete it. Let version control do it's job.

yeeted.